tests/50-live-user_notification.tests

```diff
@@ -0,0 +1,11 @@
+#
+# libseccomp regression test automation data
+#
+# Copyright Cisco Systems 2019
+# Author: Tycho Andersen <tycho@tycho.ws>
+#
+
+test type: live
+
+# Testname	API	Result
+50-live-user_notification	5	USER_NOTIF
```

Comment:
Since you need to respin this patch anyway, you might as well rebase to the current master branch and resolve the test number conflict (we now have a test 50).

Reply:
You also should have a Python version of this test.